.github/workflows/payment-reconciliations-cd.yml

```diff
@@ -3,52 +3,90 @@ name: Payment Reconciliations Queue CD
 on:
   push:
     branches:
-      - development
-      - master
+      - main
     paths:
-      - "queue_services/payment-reconciliations/**"
+      - "queue_services/payment-reconciliation/**"
```

Comment:
missing **s**, it's payment-reconciliations

Reply:
Fixed